common/core/src/main/java/com/vmware/connectors/common/payloads/response/Card.java

```diff
@@ -353,16 +367,68 @@ public Builder addTag(String tag) {
             return this;
         }
 
+        /**
+         * Add SHA-1 of card fields.
+         *
+         * @param hash SHA-1 of card header, body, and action fields.
+         * @return this Builder instance, for method chaining
+         */
+        public Builder addHash(String hash) {
```

Comment:
Also, this should be `setHash`, not `addHash`.  The `add` methods are because they add to a collection (`addTag` adds to the tags).